deploy-blueprint/library/util/playbooks/delete.yml

```diff
@@ -0,0 +1,6 @@
+- hosts: all
+  tasks:
+    - name: Delete file
+      file:
+        state: "{{ absent }}"
```

Comment:
"absent" is not a variable, but a string value, therefore double-curly-brackets should be removed. In principle, this will fail the playbook, since variable "absent" does not exist.

Reply:
I don't think this ever gets called -- it was copied directly out of xOpera-rest-api.